open-machine-learning-jupyter-book/ml-fundamentals/regression/linear-and-polynomial-regression.md

```diff
@@ -356,17 +356,6 @@ The code below shows how we can one-hot encode a variety:
 pd.get_dummies(new_pumpkins['Variety'])
 ```
 
- ID | FAIRYTALE | MINIATURE | MIXED HEIRLOOM VARIETIES | PIE TYPE
```

Comment:
You just simplify remove this. Can above code generate such result?

Reply:
yes, it can generate a such result, I have tried.